### PR TITLE
show single review when clicked

### PR DESCRIPTION
**Pull request by tyr-ros**

### CODE REVIEW

File: fe-games/src/components/Review.jsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+        className="img-container"
+        src={currentReview.review_img_url}
+        alt={currentReview.title}
+      />
+      <p>{currentReview.title}</p>
+      <p>{currentReview.category}</p>
+      <p>{currentReview.designer}</p>
+      <p>{currentReview.owner}</p>
+      <p>{currentReview.review_body}</p>
+      <p>{currentReview.votes}</p>
```
Comment (Jodiech) [inline fe-games/src/components/Review.jsx]: I'd recommend adding 'Votes:' before the number, it's not clear what the number represents here.

File: fe-games/src/components/Review.jsx [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+import { dateConverter } from "./utils";
+import { useParams } from "react-router-dom";
+
+export function Review() {
+  const [currentReview, setCurrentReview] = useState([]);
+  const [isLoading, setLoading] = useState(true)
+
+  const { review_id } = useParams();
+
+  console.log(review_id);
```
Comment (Jodiech) [inline fe-games/src/components/Review.jsx]: Please remove any console logs 👍

File: fe-games/src/components/ReviewsList.jsx [hunk 3]
```diff
@@ -29,7 +31,7 @@ if (isLoading){
                 src={review.review_img_url}
                 alt={review.title}
               />
-              <p className="title">{review.title}</p>
+              <Link to={`/review/${review.review_id}`}> <p className="title"> {review.title}</p></Link>
```
Comment (Jodiech) [inline fe-games/src/components/ReviewsList.jsx:34]: Do you want your title to be a <p> tag? Would a H1 be more suitable for this? 😄

File: fe-games/src/index.css [hunk 4]
```diff
@@ -91,19 +91,6 @@ justify-content: left;
...
-  border-radius: 50%;
-  width: 130px;
-  height: 130px;
-  animation: spin 2s linear infinite;
-}
-
-@keyframes spin {
-  0%  { transform: rotate(0deg); }
-  100% { transform: rotate(360deg); }
-}
```
Comment (Jodiech) [inline fe-games/src/index.css:106]: This app should be designed for mobile first. When you view this in the browser, it isn't responsive and the text boxes are out of line. The text colour is difficult to read, please change this so it provides a better experience for the user 👍

**Review by Jodiech — COMMENTED** [review]
Well done on this ticket 👍 there are a few comments dotted around for you to look at. Please make some changes and resubmit when you are ready. 🌻

**Review by Jodiech — COMMENTED** [review]
Well done on this ticket 👍 there are a few comments dotted around for you to look at. Please make some changes and resubmit when you are ready. 🌻

File: fe-games/src/components/Review.jsx [hunk 5]
```diff
@@ -5,36 +5,36 @@ import { useParams } from "react-router-dom";
...
+      setLoading(false);
       setCurrentReview(review);
     });
   }, [review_id]);
 
-  if (isLoading){
-    return <span className="loader"></span>
+  if (isLoading) {
+    return <span className="loader"></span>;
   }
```
Comment (leekli) [inline fe-games/src/components/Review.jsx]: I can see you have implemented an isLoading pattern, however when the articles are still loading, nothing shows on my screen, it's just blank apart from the header and nav bar. There's no indication to me, as a user, that data is loading? Maybe add a `<p>` tag with some text also?

**Review by leekli — CHANGES_REQUESTED** [review]
Hi Rebecca, 

Thanks for implementing the changes which Jodie suggested last time. I have a couple of extra ones:

- The loading pattern isn't quite displaying anything when the API goes to fetch a single article, please have another look at this. There's no indication to me, as a user, that the page is loading but I can see you've implemented some logic for it.
- The text on a single article (and the all articles list) is white, on a very light background colour; essentially it's almost impossible to read, please can you have another look at this.
- I can see you've made some changes to make the app more responsive, but it still doesn't quite work in mobile view. If you're on Chrome, if you open the inspect/dev tools, you can put the view into mobile mode and see what I mean. I'll send you a video via slack DM now so that you can see what I see... it's mostly that the image doesn't adjust to fix in the box, it starts overlapping everything if the screen goes to mobile view.... it looks fine in tablet/laptop view though. Please do take another look at the mobile view.

Please review the feedback, implement the requested changes, and submit a new PR for this ticket once complete.

File: fe-games/src/components/Review.jsx [hunk 6]
```diff
@@ -0,0 +1,40 @@
+import { useState, useEffect } from "react";
+import { fetchReviewById } from "./api";
+import { dateConverter } from "./utils";
+import { useParams } from "react-router-dom";
+
+export function Review() {
+  const [currentReview, setCurrentReview] = useState([]);
```
Comment (Keb84) [inline fe-games/src/components/Review.jsx:7]: Have a think about what data type you're working with here 👀 what are you expecting a single review to look like?

**Review by Keb84 — APPROVED** [review]
Great work on ticket 5 🚀 have a look at my comments and merge when ready 🙂

**Pull request merged**